### ISSUE TITLE
Clicking noise when using the Vocals/Accompaniment tool

### ISSUE THREAD

**Issue by lSevenLl** [RVC-Project/Retrieval-based-Voice-Conversion-WebUI#607]
I used it first on the collab it worked nicely, when I tried using the web UI locally. I do not know why, but I got the output instrumental to have a clicking noise throughout the whole audio. Not only that, but I've tried using multiple models, but it doesn't work anymore :(. https://drive.google.com/file/d/1FrqIN-gCvpzM41taSN9v3A65pmdaAcfU/view?usp=sharing

**Comment by TripleKiller666** [comment]
It's best to share some examples and Screenshots for stuff like this.
An Audiofile to listen to, so we can see if the Problem is only on your End.
A Screenshot of the Settings you use, or the Command Window, so we can see the possible Errors you encounter.

Just saying "it doesn't work anymore" isn't really helpful, right? ;D

**Comment by lSevenLl** [comment]
Good point. Here's the link of the mp3 excerpt https://drive.google.com/file/d/1FrqIN-gCvpzM41taSN9v3A65pmdaAcfU/view?usp=sharing

**Comment by Adenine14** [comment]
Hey, I am having the exact same issue as above. The popping/crackling noise is the same no matter what model I use to extract intrumentals. 
![Screenshot 2023-09-16 032159](https://github.com/RVC-Project/Retrieval-based-Voice-Conversion-WebUI/assets/81045017/c33578e1-e5dd-4f8e-9112-5bd266001741)
![Screenshot 2023-09-16 032231](https://github.com/RVC-Project/Retrieval-based-Voice-Conversion-WebUI/assets/81045017/7d1304bd-42f5-49c5-91ab-93e20cb1fc3f)

**Comment by Adenine14** [comment]
Well after a bit of fussing, I found the problem. The sound file is way too loud to be used in rvc. I dropped it into audacity and was able to lower the volume quite a bit.
![Screenshot 2023-09-16 040855](https://github.com/RVC-Project/Retrieval-based-Voice-Conversion-WebUI/assets/81045017/9d784532-87b5-484d-bc27-c0a2b51e5e0d)
This is what it looks like after exporting from audacity. This completely fixed the problem.
![Screenshot 2023-09-16 041128](https://github.com/RVC-Project/Retrieval-based-Voice-Conversion-WebUI/assets/81045017/f985e685-5128-49b3-909d-a3a9595375ff)
